_includes/tutorials/console-consumer-produer-basic/kafka/markup/dev/consume-topic-from-beginning.adoc

```diff
@@ -1,4 +1,4 @@
-Next let's open up a console consumer again, but this time you'll ready everthing that your producer has sent to the topic. you created in the previous step.
+Next let's open up a console consumer again, but this time you'll ready everything that your producer has sent to the topic. you created in the previous step.
```

Comment:
@gAmUssA Hi Viktor, can you help with this one line here? I saw this one line and wanted to see what we were meaning to say in the line? Looks we're missing a word or two. I can update it here and I'll remove the extra "." as well.

Reply:
Looks like . is indeed redundant but whole sentence is a pretty big and difficult to read. Feel free to split it.